### ISSUE TITLE
Poster session functionality

### ISSUE THREAD

**Issue by dionzand** [pretalx/pretalx#1692]
### Problem and impact

Currently there is no dedicated functionality/plugin for organizing a poster session. Scheduling a poster session as separate "talks" is not ideal, since poster sessions are in parallel.

### Proposed solution / feature

It would be nice to have dedicated functionality to schedule a poster session, where multiple "talks" can occur simultaneously. Or to have a "room" that is dedicated to a poster session, and you can even maybe create a floorplan.

### Context or examples

Poster sessions are often an important part of (scientific) conferences. Maybe this functionality can even be expanded to "exhibition stands", which are also often part of large scientific conferences.

**Comment by rixx** [comment]
This would definitely be cool to have in pretalx, but would also a fair bit of effort to add. Whoever implements this will have to consider the following:

- How to model this on the session side: Is the "poster session" a functionality of the submission object, or of its session type, or something else? I believe it ought to be the session type, as it's already used to determine session duration, and fits in there.
- How to model this on the schedule / talk slot side. Are the poster session parts regular talk slots (which would be good for a lot of interfaces, like showing a session's location and date, but bad because it breaks expectations e.g. re: uniqueness of talk slot IDs), or is it a new object type like a ChildTalkSlot? Possibly a TalkSlot with type="dependent" could work, and we just never render those in APIs that we would break this way 
- Make sure exports and APIs still work. APIs in particular do not expect nested sessions, so figuring this out in a backwards-compatible way that isn't painful to use going forward will take some doing
- Consider permissions – with how pretalx works, reviewers could have access to only some posters, but not the superordinate session, and we'd have to make sure that's honoured
- UI considerations: how do we show a poster session in the schedule? The session detail page should not be a problem, but the overall schedule will prove challenging
- UI considerations: how do posters get added to and moved between poster sessions? Do we have this functionality in the schedule editor, or just in the individual poster form? Can you then re-schedule a poster? etc etc
- Make sure changes show correctly in the schedule changelog, which means that TalkSlots will have to have some knowledge of their child slots, and can't just directly refer to a session

I'm sure this is not all of it, but that's what came to mind lately. Happy to work on this on a sponsored/commissioned basis, or review a PR, as I don't think I'll find the time to prioritise this any time soon otherwise.

**Comment by johanvdw** [comment]
At FOSDEM we have a call for stands, which has very similar requirements. We have used pretalx for this the first time this year (as a separate conference), and we were happy that we could reuse the same review interface as for talks.

I am not really sure that a hardcoded link to schedule is needed. If we can just have a list of accepted items in a certain category a schedule item could link to that.  In some cases, posters or stands are available the whole conference, so adding it as a schedule item is  not needed.

As a minor remark, while adding proposals, currently 'speakers' is hardcoded in a number of places (such as add a speaker). It would be nice if this could be overwritten if you already know that your CfP is not for speakers. I'm planning to make a PR for that.

**Comment by rixx** [comment]
> As a minor remark, while adding proposals, currently 'speakers' is hardcoded in a number of places (such as add a speaker). It would be nice if this could be overwritten if you already know that your CfP is not for speakers. I'm planning to make a PR for that.

I think I’d generally prefer to find a neutral word rather than allowing organiser-level overrides, as those would be a pain in i18n. I did play with "presenter" one time, but for some reason wasn’t entirely convinced. Once we build #527, we could even get away with just "people" …

**Comment by RemDelaporteMathurin** [comment]
@rixx do you know if this feature is still planned or if there has been some progress towards it?

**Comment by rixx** [comment]
@RemDelaporteMathurin The issue is not planned, but it is accepted (ie: I agree it would be cool to have in core, but there is no timeline).

There is at least one structural, data-model level issue that needs solving before this feature can be implemented, connected to how pretalx currently handles multi-slot sessions.

**Comment by RemDelaporteMathurin** [comment]
@rixx although I've never contributed to pretalx yet I'd be happy to help with this as we are using pretalx for [OSSFE 2027](https://ossfe.org/OSSFE_2027/) (and we'll have poster sessions!)

What's the best way to assist in this?
